File: src/Mobile/ReportsApp/ReportsApp/ReportsApp/Models/Report.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ReportsApp.Models
{
    public class Report
    {
        public string Name { get; set; }
        public string Address { get; set; }
        public License License { get; set; }
        public DateTime Date { get; set; }
        public bool IsClosed { get; set; }
        public string Title { get; set; }
        public string FormName { get; set; }
        public string RemarksLabelText { get; set; }
        public List<Signature> Signatures { get; set; }
        public List<Note> Notes { get; set; }
        public List<CheckList> CheckList { get; set; }
        public List<object> PhotoRecords { get; set; }
        public bool Completed { get; set; }
        public bool NotCompleted => !Completed;
        public bool IsNotClosed => !IsClosed;
        public int Id { get; set; }
    }
}

File: src/ClientApp/types/Reports/Models/Report.ts
import { PhotoRecord } from './PhotoRecord'
import { Note } from './Note'
import { EMALicense } from './EMALicense'
import { Signature, CheckList } from '~/types'

export interface Report {
    id:number;
    name: string;
    address: string;
    license: EMALicense;
    date: string;
    isClosed: boolean;
    title: string;
    formName: string;
    remarksLabelText: string;
    signatures: Signature[];
    notes: Note[];
    checkList: CheckList[];
    photoRecords: PhotoRecord[];
    completed: boolean;
    reportConfigurationId: number;
}
